doc/adoc/user_guide.adoc

```diff
@@ -163,6 +163,11 @@ The product must be specified with the triplet PROD_NAME/VERSION/ARCH, for examp
 [listing]
 migration_product: 'SLES/15/x86_64'
 
+or 
+
+migration_product: 'SLES/15.2/x86_64' (Migrate to SLES-15-SP2)
+migration_product: 'SLES/15.3/x86_64' (Migrate to SLES-15-SP3)
```

Comment:
And now you'd have to also show how this is done for SLES For SAL and then for SLE HPC and then for potentially other products.

I think we should give 1 example and then explain to the user how to compose the triplet. Yes, explaining how to compose the triplet is a little more difficult as the system itself does not have any of this information but we should be able to explain this in a way that makes sens without having to list every product and every possible version. Further listing a specific target in the doc may give the impression that the listed target is supported.